mantle/kola/tests/ignition/luks.go

```diff
@@ -176,6 +187,51 @@ func runTest(c cluster.TestCluster, tpm2 bool, threshold int, killTangAfterFirst
 	ut.LUKSSanityTest(c, tangd, m, tpm2, killTangAfterFirstBoot, rootPart)
 }
 
+func runCexTest(c cluster.TestCluster, cex bool) {
+	ignition := conf.Ignition(fmt.Sprintf(`{
+		"ignition": {
+			"version": "3.5.0"
+		},
+		"storage": {
+			"luks": [
+				{
+					"name": "root",
+					"device": "/dev/disk/by-label/root",
+					"cex": {
+						"enabled": %v,
+					},
+					"label": "root",
+					"wipeVolume": true
+				}
+			],
+			"filesystems": [
+				{
+					"device": "/dev/mapper/root",
+					"format": "xfs",
+					"wipeFilesystem": true,
+					"label": "root"
+				}
+			]
+		}
+	}`, true))
+
+	opts := platform.MachineOptions{
+		MinMemory: 8192,
+	}
+
+	rootPart := "/dev/disk/by-partlabel/root"
+
+	if coreosarch.CurrentRpmArch() == "s390x" {
+		rootPart = "/dev/disk/by-id/virtio-primary-disk-part4"
+	}
```

Comment:
Should be able to drop this. I think this is an old workaround for partlabel issues on s390x.

Reply:
Will correct it.